projects/netgrif-components-core/src/lib/side-menu/content-components/new-case/abstract-new-case.component.ts

```diff
@@ -161,6 +169,7 @@ export abstract class AbstractNewCaseComponent implements OnDestroy {
                                 data: (response.outcome as CreateCaseEventOutcome).aCase
                             });
                         } else if (!!response.error) {
+                            this.loadingSubmit.off();
```

Comment:
add this also to error part of subscribe

Reply:
Done